packages/react-storage/src/components/StorageBrowser/context/controls/locationActions/defaults.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { IconElement } from '../../elements/IconElement';
+
+import { TaskActionOutput, TaskActionInput } from '../../types';
+import { LocationAction } from './types';
+
+const handler = (_: TaskActionInput): Promise<TaskActionOutput> =>
+  Promise.resolve({ result: undefined });
+
+const CREATE_FOLDER: LocationAction = {
+  handler,
+  options: {
+    displayName: 'Create Folder',
+    disable: (items) => !!items.length,
+    hide: (permission) => permission === 'READ',
+    icon: <IconElement variant="create-folder" />,
+  },
+};
+
+const UPLOAD_FOLDER: LocationAction = {
+  handler,
+  options: {
+    disable: (items) => !!items.length,
+    displayName: 'Upload Folder',
+    hide: (permission) => permission === 'READ',
+    icon: <IconElement variant="upload-folder" />,
+    selectionData: 'folder',
+  },
+};
+
+const UPLOAD_FILES: LocationAction = {
+  handler,
+  options: {
+    disable: (items) => !!items.length,
+    displayName: 'Upload File',
+    hide: (permission) => permission === 'READ',
+    icon: <IconElement variant="upload-file" />,
+    selectionData: ['file', '.png'],
```

Comment:
What's the reason for having `.png` as the default selection data?

Reply:
Just a typo from testing :)